script/saleae

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env ruby
+# Usage: script/saleae /path/to/captured_data.csv
+#        script/saleae data.csv | jq '.[] | .message'
```

Comment:
Just realized I forgot to update the `.message` to be `.bits`... gonna commit directly to master :grin: